src/models/language.rs

```diff
@@ -86,6 +86,13 @@ impl PiranhaLanguage {
     parser
   }
 
+  pub(crate) fn can_parse(&self, de: &jwalk::DirEntry<((), ())>) -> bool {
+    de.path()
+      .extension()
+      .and_then(|e| e.to_str().filter(|x| x.eq(&self.name())))
```

Comment:
Probably a nit for a separate PR, but... are using the field `name` to refer to the string representation of the language (i.e. `"kotlin"`) or the extension (i.e. `"kt")? Seems like a weird choice for "name" to be the second...